.travis.yml

```diff
@@ -30,14 +30,10 @@ matrix:
     - env: NODE_VERSION="lts/*" TEST_SUITE=performance
 
 before_install:
-  # start couchdb 2.x docker instance
-  - docker run -d -p 5984:5984 --name couch couchdb:2
-  - echo "Starting CouchDB 2.x"
-  - until nc -z localhost 5984; do sleep 1; done
-  - echo "CouchDB Started"
+  - ./scripts/travis/couch-start
```

Comment:
We have a `/scripts/ci` with stuff like this - I reckon merge `/scripts/travis` and `/scripts/ci` one way or the other.

Reply:
Of course.  I prefer `travis` to `ci` for clarity, but I don't think changing that within this PR is useful.

Reply:
Actually... I can't find the `/scripts/ci` directory on master: https://github.com/medic/medic-webapp/tree/master/scripts